cypress/utils/index.js

```diff
@@ -1,19 +1,30 @@
 import Web3 from "web3";
 import PrivateKeyProvider from "truffle-privatekey-provider";
 
+// this is exactly the same as ganache.sh file in localnode
+const ACCOUNT_1_PRIV = "0x47be2b1589bb515b76b47c514be96b23cd60ee37e81d63c2ae9c92f7d7667e1a";
```

Comment:
Why do we have 1 and 3 but we don't have 2?

Reply:
it's corresponding to list hardcoded in localnode/ganache.sh I used account3 here since it was least used in the past 🤷‍♂️